packages/clerk-js/src/utils/injectedWeb3Providers.ts

```diff
@@ -52,6 +52,12 @@ class InjectedWeb3Providers {
   }
 
   get = (provider: InjectedWeb3Provider) => {
+    // In case there is a single injected provider, use that directly.
+    // This is needed in order to support other compatible Web3 Wallets (e.g. Rabby Wallet)
+    // and maintain the previous behavior
+    if (this.#providers.length === 1) {
+      return this.#providers[0].provider;
+    }
     return this.#providers.find(p => p.info.name === this.#providerIdMap[provider])?.provider;
   };
```

Comment:
Just confirming here: If a user has a single wallet installed that is not metamask we accept it. But if the have Metamask and Rabby only metamask will be accepted ?

Reply:
That's correct and hopefully that will solve the issues that some users had after this change